platform/settings.cpp

```diff
@@ -24,6 +24,10 @@ std::string_view kMeasurementUnits = "Units";
 std::string_view kMapLanguageCode = "MapLanguageCode";
 std::string_view kDeveloperMode = "DeveloperMode";
 
+std::string_view kDonateUrl = "DonateUrl";
+std::string_view kPlacePageProductsCloseTime = "PlacePageProductsCloseTime";
+std::string_view kPlacePageProductsSelectedTime = "PlacePageProductsSelectedTime";
```

Comment:
Probably last 2 options are needed only in framework.cpp, no?

Reply:
Yes. But I suppose that it is better to store all the sittings keys in settings to avoid key duplication and unessesary string literals usage

Reply:
We already have a dozen of keys declared in framework.cpp